rumqttd/src/server/broker.rs

```diff
@@ -17,8 +13,15 @@ use flume::{RecvError, SendError, Sender};
 use std::net::{IpAddr, Ipv4Addr, SocketAddr};
 use std::sync::Arc;
 use tracing::{error, field, info, Instrument};
-#[cfg(feature = "websockets")]
-use websocket_codec::MessageCodec;
+
+#[cfg(feature = "websocket")]
+use async_tungstenite::tokio::accept_hdr_async;
+#[cfg(feature = "websocket")]
+use async_tungstenite::tungstenite::handshake::server::Callback;
+#[cfg(feature = "websocket")]
+use async_tungstenite::tungstenite::handshake::server::*;
```

Comment:
can we explicitly import the things we need instead on `*`  ? 
```suggestion
#[cfg(feature = "websocket")]
use async_tungstenite::tungstenite::handshake::server::{
    Callback, ErrorResponse, Request, Response,
};
```